src/hssm/plotting/model_cartoon.py

```diff
@@ -0,0 +1,1985 @@
+            t_s=t_s,
+            color=color_pp,
+            zorder_cnt=z_cnt + 1,
+        )
+
+    # Add in trajectories
+    if n_trajectories > 0:
+        _add_trajectories(
+            axis=axis,
+            sample=sim_out_traj,
+            t_s=t_s,
+            delta_t_graph=delta_t_model,
+            n_trajectories=n_trajectories,
+            alpha_trajectories=alpha_trajectories,
+            **kwargs,
+        )
+    return axis
+
+
+# AF-TODO: Add documentation for this function
+def _add_trajectories(
+    axis: Axes,
+    sample: dict,
+    t_s: np.ndarray,
+    delta_t_graph: float = 0.01,
+    n_trajectories: int = 10,
+    highlight_trajectory_rt_choice: bool = True,
+    markersize_trajectory_rt_choice: float | int = 50,
+    markertype_trajectory_rt_choice: str = "*",
+    markercolor_trajectory_rt_choice: str | int = "red",
+    linewidth_trajectories: float | int = 1,
+    alpha_trajectories: float | int = 0.5,
+    color_trajectories: str = "black",
+    **kwargs,
+):
+    """Add simulated decision trajectories to a given matplotlib axis.
+
+    Parameters
+    ----------
+    axis : matplotlib.axes.Axes
+        The axis to add the trajectories to.
+    sample : dict
+        Dictionary containing simulation data including trajectories and metadata.
+    t_s : numpy.ndarray
+        Array of timepoints for plotting.
+    delta_t_graph : float, optional
+        Time step size for plotting, by default 0.01.
+    n_trajectories : int, optional
+        Number of trajectories to plot, by default 10.
+    highlight_trajectory_rt_choice : bool, optional
+        Whether to highlight the response time and choice with a marker, by default
+        True.
+    markersize_trajectory_rt_choice : float or int, optional
+        Size of marker for response time/choice, by default 50.
+    markertype_trajectory_rt_choice : str, optional
+        Marker style for response time/choice, by default "*".
+    markercolor_trajectory_rt_choice : str, int, list or dict, optional
+        Color(s) for response time/choice markers. Can be a single color,
+        list of colors, or dict mapping choices to colors. By default "red".
+    linewidth_trajectories : float or int, optional
+        Line width for trajectories, by default 1.
+    alpha_trajectories : float or int, optional
+        Opacity of trajectories, by default 0.5.
+    color_trajectories : str, list or dict, optional
+        Color(s) for trajectories. Can be a single color, list of colors,
+        or dict mapping choices to colors. By default "black".
+    **kwargs
+        Additional keyword arguments passed to plotting functions.
+    """
+    # Check markercolor type
+    if isinstance(markercolor_trajectory_rt_choice, str):
+        markercolor_trajectory_rt_choice_dict = {}
+        for value_ in sample[0]["metadata"]["possible_choices"]:
+            markercolor_trajectory_rt_choice_dict[value_] = (
+                markercolor_trajectory_rt_choice
+            )
```

Comment:
Use a dict comprehension.
```suggestion
        markercolor_trajectory_rt_choice_dict = {
            value_: markercolor_trajectory_rt_choice
            for value_ in sample[0]["metadata"]["possible_choices"]
        }
```